readme.md

```diff
@@ -283,13 +283,18 @@ install the dependencies:
     echo 'amti' > .python-version
     pip install -r requirements.txt
 
-Then, make sure that you have the proper environment variables set in
+Then, make sure that you have the proper AWS environment variables set in
```

Comment:
Perhaps we should link `AWS environment variables` to the most relevant bit of documentation? For example [this page](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).